plugins/outputs/newrelic/newrelic.go

```diff
@@ -4,12 +4,13 @@ package newrelic
 import (
 	"context"
 	"fmt"
+	"github.com/influxdata/telegraf/plugins/outputs"
```

Comment:
Moving this line here is not correct. We like to group the influx imports below.

Reply:
Done